doc/en/components/grids/_shared/multi-column-headers.md

```diff
@@ -479,6 +561,17 @@ public columnHeaderTemplate = (ctx: IgcCellTemplateContext) => {
 }
 ```
 
+```tsx
+function columnHeaderTemplate(e: { dataContext: IgrColumnTemplateContext }) {
+    const column = e.dataContext.column as IgrColumnGroup;
+    return (
+      <span onClick={(evt) => this.onClick(evt)}>
```

Comment:
I think the `this` usage is not applicable to the function syntax. Other than that it looks good.